[DOCS] Threat Intelligence - Indicators Page and Indicator Details (#2526)
elastic/security-docs: docs/events/indicators-of-compromise.asciidoc

[[indicators-of-compromise]]
= Indicators of compromise

> The Intelligence page collects data from enabled threat intelligence feeds and provides a centralized view of indicators. This guide defines threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.

Author: Making some minor capitalization edits:

```suggestion
The Intelligence page makes it easy to access and analyze your threat intelligence data by centralizing Indicators of Compromise (IoCs) from enabled threat intelligence feeds and providers. This guide defines threat intelligence and IoCs, helps you get started using the Intelligence page, and explains how to work with indicators.
```

[role="screenshot"]
image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]

[discrete]
[[ti-indicators]]
== Threat intelligence and indicators

> Threat intelligence is a research function that helps an organization make informed security decisions and actions. An organization's security operations center (SOC) team usually manages threat intelligence, which helps them understand current and emerging threats and recommend actions that will prevent, stop, and mitigate security threats.

Reviewer: ```suggestion
Threat intelligence is a research function that is responsible for understanding current and emerging threats, and recommending appropriate actions to strengthen the organization's security posture. Producing actionable threat Intelligence requires aggregating, analyzing, and synthesizing threat and vulnerability data from various sources and conducting proactive investigations.
```

Reviewer: I feel like my second sentence can be better worded

Author: @dhru42 thanks for the additional content! Here's a potential revision:

```suggestion
Threat intelligence is a research function that is responsible for understanding current and emerging threats, and recommending appropriate actions to strengthen the organization's security posture. Threat intelligence must be actionable to be useful. Producing it requires gathering, analyzing, and synthesizing various threat and vulnerability data sources and conducting proactive investigations.
```

> If you are unfamiliar with threat intelligence, think of it as a research function that helps an organization make informed decisions when taking actions to protect itself. An organization's security operation center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can recommend actions that will prevent, stop, and mitigate threats to an organization.

Reviewer 2: ```suggestion
If you are unfamiliar with threat intelligence, think of it as research that helps an organization make informed security decisions. An organization's security operations center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can prevent, stop, and mitigate security threats.
```

Author: Thanks for these suggestions, @joepeeples! I agree that the target audience for these docs probably already understands what threat intelligence (TI) and IoCs are. In fact, I debated explaining the terms here or adding them our [terms glossary](https://www.elastic.co/guide/en/security/master/detection-engine-overview.html#det-engine-terminology) and linking to that section. I mainly wanted to see if this background info flowed well with the rest of the topic and provided the necessary context for understanding some of the more jargony terms that are used in the instructions like "threat intelligence", "threat intelligence feeds" (I probably need to add this to the terms glossary) and "indicators" (which I believe is a spelt-out abbreviated form of IoCs?). Anyways, I don't have strong feelings about keeping the content here or moving it elsewhere if everyone thinks the content is more appropriate somewhere else.

Reviewer 3: @nastasha-solomon Thanks for the extra background. Like I said, I think this is still good content in general, and it does accomplish your goal of providing context for the terms. But we can present this information without making it sound like readers are learning about threat intel for the first time — even if they are!

> An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information that represents a known malicious threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.

Reviewer 2: ```suggestion
An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information associated with a known threat or vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
```

Reviewer 3: A few tweaks to @benironside's edit. I kinda think "reported" is good to include since a lot of threat intel seems to depend on reporting of info:
```suggestion
An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
```

Author: I've seen indicators of compromise written both ways but am totally fine using the all lowercase form and treating it like a common noun

[discrete]
[[setup-intelligence-page]]
== Set up the Intelligence page

Install threat intelligence integrations to add indicators to the Intelligence page.

IMPORTANT: Before enabling a threat intelligence feed, ensure you have installed a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the hosts you want to monitor.

. Choose one of the following:
.. From the {security-app} main menu, click the Intelligence page, then click *Add Integrations*.

> .. From the {kib} main menu and click *Add integrations*. Scroll down list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.

Reviewer 2: ```suggestion
.. From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
```

> .. From the {security-app} main menu, click the Intelligence page, then click *Add Integrations*.
> .. From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.

Reviewer 3: Change to bullets, since these are separate options, not sequential steps. I _think_ this should work (per [Asciidoc docs](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/#nested-ordered-list)) -- also tightening up menu options in the first bullet:
```suggestion
* From the {security-app} main menu, click *Intelligence* -> *Add Integrations*.
* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
```

Author: Oh yeah, good catch - thanks!

> . Select a threat intelligence integration, then complete the installation steps.

Reviewer 3: Maybe something like "complete the on-screen installation steps"? I don't know if "on-screen" is the right word, but something to indicate that the UI will guide the user. When I first read this, I thought it was referring to the docs.

Author: How about this:

```suggestion
. Select a threat intelligence integration, then complete the integration's guided installation.
```

Reviewer 3: @nastasha-solomon nailed it, LGTM!

+

> NOTE: For more information about required field mappings, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.

Reviewer 4: ```suggestion
NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
```

> NOTE: Refer to the Threat intelligence section of the https://docs.elastic.co/integrations[Elastic integration documentation] for more information about required field mappings.

Reviewer 3: This was a little unclear at first, partly because "Threat intelligence" is only a menu heading on the target page. It might help to be more explicit about this, because I was confused when I first landed on the page.
```suggestion
NOTE: For more information about required field mappings, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
```

Author: ^This works and sorry for the confusion. I don't think its actually possible to link to the Threat Intelligence section in the integration docs (yet) 😅  Though, do let me know if there's a way. Otherwise, I like the suggestion you made to tell users that they could search for the TI integration they're installing.

Reviewer 3: @nastasha-solomon Yeah I tried to find a way to link to that section too, but couldn't find anything. "Search for it" seemed easier to say than trying to explain how to use the nav menu.

. Return to the Intelligence page in {elastic-sec}. Refresh the page if indicator data isn't displaying.

[discrete]
[[intelligence-page-ui]]
== Intelligence page UI

After you add indicators to the Intelligence page, you can <<examine-indicator-details,examine indicators>> and search, filter, and take actions on indicator data. Indicators are also visualized in the Trend view, which shows the occurrence of indicators over time.

[role="screenshot"]
image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]

[discrete]
[[examine-indicator-details]]
=== Examine indicator details

> Learn more about an indicator by clicking the *View details* button and opening the Indicators details flyout. The flyout contains these informational tabs:

Reviewer 2: ```suggestion
Learn more about an indicator by clicking the *View details* button and opening the Indicator details flyout. The flyout contains these informational tabs:
```

> Learn more about an indicator by clicking the *View details* button and opening the Indicator details flyout. The flyout contains these informational tabs:

Reviewer 3: ```suggestion
Learn more about an indicator by clicking *View details* and opening the Indicator details flyout. The flyout contains these informational tabs:
```

> * *Overview*: Offers a summary of the indicator. Provides the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.

Reviewer 2: ```suggestion
* *Overview*: Summarizes the indicator. Displays the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

Reviewer 3: To match the form of the other bullet items, the first sentence should actually be verb-less. After that, no preference on what verb the next sentence uses.
```suggestion
* *Overview*: A summary of the indicator. Provides the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

+

> NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The *TLP Marking* field will be empty if the data is not available.

Reviewer 4: It's true not only for TLP Marking but for Confidence as well

> * *Table*: The indicator data in table format. Data is organized into field-value pairs.

Reviewer 3: Nit: It feels unnecessary to mention the field-value pairs detail; it seems pretty obvious, especially if the table has headings saying "Field" and "Value" (the screenshot shows the Overview tab but I assume Table tab is similar?)

Author: Good point and yeah the **Table** tab is nearly identical.

Author: ```suggestion
* *Table*: The indicator data in table format. 
```

[[indicators-of-compromise]]
= Indicators of compromise

> The Intelligence page collects data from enabled threat intelligence feeds and provides a centralized view of indicators. This guide defines threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.

Author: Ah, I see what you're saying and that's a great, very forward-thinking way of organizing the docs. I have no objections to changing all references to the page from "Intelligence page" to "Indicators page" if that's what we end up deciding.

> The Intelligence page collects data from enabled threat intelligence feeds and provides a centralized view of indicators. This guide defines threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.

Reviewer 4: makes sense to me

[role="screenshot"]
image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]

.Requirements
[sidebar]
--
You must have _one_ of the following installed on the hosts you want to monitor:

* *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's {fleet-guide}/view-elastic-agent-status.html[status] is `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
--

[discrete]
[[ti-indicators]]
== Threat intelligence and indicators
Threat intelligence is a research function that helps analyze current and emerging threats, and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.

> An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.

Reviewer 5: I feel like we're repeating ourselves with what's on [line 4](https://github.com/elastic/security-docs/pull/2526/files#diff-b631a6767b67ba08f4fd47f195fd4193da87a562f6d259ed16aee534bdaf483aR4). This could just be:
`An IoC is a piece of information associated with a known threat or reported vulnerability.`

[discrete]
[[setup-intelligence-page]]

> == Set up the Intelligence page

Reviewer 6: Going by the UI here. 

```suggestion
== Set up the Indicators page
```

> Install threat intelligence integrations to add indicators to the Intelligence page.

Reviewer 6: Do they need to install more than one? If not, we might want to rephrase this.

Author: Ah, good catch. Thanks!

```suggestion
Install a threat intelligence integration to add indicators to the Intelligence page.
```

> NOTE: If you've already installed threat intelligence integrations, you do not need to complete the following steps. Indicator data will automatically display within the Indicators table.

Reviewer 6: ```suggestion
NOTE: If you've already installed a threat intelligence integration, you don't need to complete the following steps. Indicator data will automatically display in the Indicators table.
```

. Choose one of the following:

> * From the {security-app} main menu, click *Intelligence* -> *Add Integrations*.

Reviewer 6: ```suggestion
* From the {security-app} main menu, go to *Intelligence* -> *Add Integrations*.
```

> * From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.

Reviewer 6: ```suggestion
* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter by threat intelligence integrations.
```

> . Select a threat intelligence integration, then complete the integration's guided installation.

Reviewer 6: I think there was one integration that @maxcold recommended users not select. I'll check Slack, but that's where it was mentioned.

+
NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.

. Return to the Intelligence page in {elastic-sec}. Refresh the page if indicator data isn't displaying.

[discrete]
[[troubleshoot-intel-page]]
=== Troubleshooting

> If indicator data is not appearing in the Indicators table after you installed threat intelligence integrations:

Reviewer 6: Same question about installing multiple integrations.

Author: ```suggestion
If indicator data is not appearing in the Indicators table after you installed a threat intelligence integration:
```

> * Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based the way you're collecting indicator data:

Reviewer 6: I feel like `index holding` needs a hyphen? `index-holding` 🤔 but feel free to omit if I'm wrong. 
```suggestion
* Verify that the index-holding indicator documents are included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index-holding indicator documents will differ depending on how you're collecting indicator data:
```

Author: Is this better/more clear, @jmikell821 :

* Verify that the indicators' index is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index will differ depending on how you're collecting indicator data:

> * Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based the way you're collecting indicator data:

Reviewer 4: ```suggestion
* Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based on the way you're collecting indicator data:
```

** *{agent} integrations* - If you are collecting indicator data from {agent} threat intelligence integrations, indicator documents are stored in the `logs_ti*` index pattern.
** *{filebeat} integrations* - If you are collecting indicator data from {filebeat} threat intelligence integrations, indicator documents are stored in the `filebeat-*` index pattern.

> * Make sure indicator data you're ingesting is mapped to {ecs-ref}[Elastic Common Schema (ECS)].

Reviewer 6: ```suggestion
* Make sure the indicator data you're ingesting is mapped to {ecs-ref}[Elastic Common Schema (ECS)].
```

[discrete]
[[intelligence-page-ui]]
== Intelligence page UI

> After you add indicators to the Intelligence page, you can <<examine-indicator-details,examine indicators>> and search, filter, and take actions on indicator data. Indicators are also visualized in the Trend view, which shows the occurrence of indicators over time.

Reviewer 6: ```suggestion
After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the occurrence of indicators over time.
```

[role="screenshot"]
image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]

[discrete]
[[examine-indicator-details]]
=== Examine indicator details

> Learn more about an indicator by clicking *View details* and opening the Indicator details flyout. The flyout contains these informational tabs:

Reviewer 6: Adding some visibility to the industry term but I'm bot sure about nuances and where to use IoC vs indicators.  
```suggestion
Learn more about an IoC by clicking *View details*, then opening the Indicator details flyout. The flyout contains these informational tabs:
```

> * *Overview*: A summary of the indicator. Provides the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.

Reviewer 6: ```suggestion
* *Overview*: Provides a summary of the indicator, including the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

+

> NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The *TLP Marking* field will be empty if the data is not available. The *Confidence* field will also be empty if the data is not available.

Reviewer 6: ```suggestion
NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The `TLP Marking` field will be empty if the data is not available. The `Confidence` field will also be empty if the data is not available.
```

* *Table*: The indicator data in table format.
* *JSON*: The indicator data in JSON format.
+
[role="screenshot"]
image::images/indicator-details-flyout.png[Shows the Indicator details flyout, 600]

[discrete]
[[find-related-sec-events]]
== Find related security events

Investigating an indicator in <<timelines-ui,Timeline>> helps you find related security events in your environment. You can add an indicator to Timeline from the Indicators table or the Indicator details flyout.

[role="screenshot"]
image::images/indicator-in-timeline.png[Shows the results of an indicator being investigated in Timeline]

When you add an indicator to Timeline, a new Timeline opens with a pre-populated KQL query. The query contains the indicator field and value pair that you selected. Timeline also automatically includes the field and value pair of the mapped source event.

> For example, if you added the file hash indicator `threat.indicator.file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba` to Timeline, Timeline automatically updates the KQL query to include the mapped source event field and value `file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`. Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field and value pairs. 

Reviewer 6: Can we see if putting the hash values on new lines looks a little less jumbled?

[[indicators-of-compromise]]
= Indicators of compromise

> The Indicators page collects data from enabled threat intelligence feeds and provides a centralized view of indicators, also known as indicators of compromise (IoCs). This topic helps you set up the Indicators page  and explains how to work with IoCs.

Reviewer 4: I would refer to Intelligence page when talking about the specific nav menu item. The Intelligence menu item in the future is planned to be the home of multiple different capabilities related to Threat Intelligence. When we talk about the technical specifics of the only available page atm which is Indicators page, I'd refer to it as Indicators page

> The Indicators page collects data from enabled threat intelligence feeds and provides a centralized view of indicators, also known as indicators of compromise (IoCs). This topic helps you set up the Indicators page  and explains how to work with IoCs.

Reviewer 3: Thanks for explaining, @maxcold! That will help us plan for the future.

@nastasha-solomon I'd still say we need to explicitly tell readers how to find the "Indicators" page in the side nav, since we currently don't. We can revise in the future when new pages get built in the "Intelligence" section.

> The Indicators page collects data from enabled threat intelligence feeds and provides a centralized view of indicators, also known as indicators of compromise (IoCs). This topic helps you set up the Indicators page  and explains how to work with IoCs.

Author: Makes sense. I'll add the nav instructions in where needed. Thanks, both!

[role="screenshot"]
image::images/indicators-table.png[Shows the Indicators page]

.Requirements
[sidebar]
--
You must have _one_ of the following installed on the hosts you want to monitor:

> * *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's {fleet-guide}/view-elastic-agent-status.html[status] is `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.

Reviewer 3: Three links in one sentence is a lot. Seems like we could omit the cross-ref to Agent status?

Author: Yeah, good idea. The troubleshooting docs are ultimately more helpful if the agent runs into issues.

```suggestion
* *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's status is `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
```

* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
--

[discrete]
[[ti-indicators]]
== Threat intelligence and indicators

> Threat intelligence is a research function that helps analyze current and emerging threats, and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.

Reviewer 3: The first sentence seems a little muddled to me. From what I remember of the previous draft, threat intel helps _people_ analyze threats and recommend actions, not that threat intel _itself_ analyzes and recommends things. Maybe add people back in, and have them doing most of the action:
```suggestion
Threat intelligence is a research function that helps security professionals analyze current and emerging threats and recommend appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.
```

Author: I'm a fan of adding a more tangible actor in the first sentence. The one thing I will note is that "security professionals" is fairly broad. Would you recommend scaling it down to "threat intelligence analysts" @dhru42? Or, is the broader term (security professionals) sufficient?

Reviewer 3: Yeah, I kind of just threw "security professional" in there, no real preference as long as the actor is clearer.

Reviewer: I'd recommend the former version where we talk about threat intelligence at the function level because it eludes that threat intelligence analysts can use our functionality, in addition to other analysts interested in accessing the threat intelligence feed output.

Author: @dhru42 would the following work?

Threat intelligence is a research function that analyzes current and emerging threats and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.

An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.

[discrete]
[[setup-indicators-page]]
== Set up the Indicators page

Install a threat intelligence integration to add indicators to the Indicators page.

> NOTE: If you've already installed a threat intelligence integration, you don't need to complete the following steps. Indicator data will automatically display on the Indicators page.

Reviewer 3: I might be missing what this note is trying to do. The previous sentence says "install this," and then this note says "if you've already installed this, then you don't need to do the installation steps." 

That seems very obvious; if you've already installed something, then of course you can ignore the installation instructions. Is there more nuance that I'm missing?

Author: Hm, good point. The lead-in sentence already says that threat intel integrations need to be installed for the page to display data. So if users have already installed those, they can likely conclude that they've completed the setup steps. Maybe this note isn't needed after all.

. Choose one of the following:
* From the {security-app} main menu, go to *Intelligence* -> *Add Integrations*.
* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter by threat intelligence integrations.
. Select a threat intelligence integration, then complete the integration's guided installation.
+
NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.

. Return to the Indicators page in {elastic-sec}. Refresh the page if indicator data isn't displaying.

[discrete]
[[troubleshoot-indicators-page]]
=== Troubleshooting

> If indicator data is not appearing in the Indicators table after you installed a threat intelligence integration:

Reviewer 3: ```suggestion
If indicator data isn't appearing in the Indicators table after you installed a threat intelligence integration:
```

* Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based on the way you're collecting indicator data:
** *{agent} integrations* - If you are collecting indicator data from {agent} threat intelligence integrations, indicator documents are stored in the `logs_ti*` index pattern.

> ** *{filebeat} integrations* - If you are collecting indicator data from {filebeat} threat intelligence integrations, indicator documents are stored in the `filebeat-*` index pattern.

Reviewer 3: Not sure about indices "holding" documents -- maybe "storing"?

Also, the bullets and lead-in are very wordy and repetitive, making readers do so much work to get simple information. This seems perfect for a table or definition list:

> Each indicator data collection method uses a different index:
> 
> - **Elastic Agent integrations**: `logs_ti*`
> - **Filebeat integrations**: `filebeat-*`
>

Author: Yeah, storing is a good alternative. It makes sense to say that indices _store_ documents.

Author: ```suggestion
* Verify that the index storing indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index storing indicator documents will differ based on the way you're collecting indicator data:
** *{agent} integrations* - `logs_ti*` 
** *{filebeat} integrations* - `filebeat-*`
```

* Make sure the indicator data you're ingesting is mapped to {ecs-ref}[Elastic Common Schema (ECS)].

[discrete]
[[intelligence-page-ui]]

> == Intelligence page UI

Reviewer 3: "Intelligence" or "Indicators" page?

After you add indicators to the Indicators page, you can <<examine-indicator-details, examine>>, search, filter, and take action on indicator data. Indicators also appear in the Trend view, which shows the occurrence of indicators over time.

[role="screenshot"]
image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]

[discrete]
[[examine-indicator-details]]
=== Examine indicator details

> Learn more about an IoC by clicking *View details*, then opening the Indicator details flyout. The flyout contains these informational tabs:

Reviewer 3: ```suggestion
Learn more about an IoC by clicking *View details*, then opening the Indicator details flyout. The flyout contains these tabs:
```

> * *Overview*: Provides a summary of the indicator, including the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.

Reviewer 3: To match the format of the other list items
```suggestion
* *Overview*: A summary of the indicator, including the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

+

> NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The `TLP Marking` field will be empty if the data is not available. The `Confidence` field will also be empty if the data is not available.

Reviewer 3: Wouldn't any field, anywhere be empty if the data isn't available? Consider omitting the last two sentences, unless there's some other reason why this needs to be mentioned.

Reviewer 4: I think the idea was to mention that these two fields which are highlighted on the overview tab will always be empty for some feeds as they never provide it. But I agree that if we want to mention it we can do it more clear.

Reviewer 3: Ah, thanks again @maxcold! This does seem good to mention, maybe something like this:
```suggestion
NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=Introduction,shared%20with%20the%20appropriate%20audience[Traffic Light Protocol (TLP) markings]. The `TLP Marking` and `Confidence` fields will be empty if the threat intelligence feed doesn't provide that data.
```

Author: 1+ for combining the last two sentences and switching to active voice.

* *Table*: The indicator data in table format.
* *JSON*: The indicator data in JSON format.
+
[role="screenshot"]
image::images/indicator-details-flyout.png[Shows the Indicator details flyout, 600]

[discrete]
[[find-related-sec-events]]
== Find related security events

Investigating an indicator in <<timelines-ui,Timeline>> helps you find related security events in your environment. You can add an indicator to Timeline from the Indicators table or the Indicator details flyout.

[role="screenshot"]
image::images/indicator-in-timeline.png[Shows the results of an indicator being investigated in Timeline]

> When you add an indicator to Timeline, a new Timeline opens with a pre-populated KQL query. The query contains the indicator field and value pair that you selected plus the field and value pair of the mapped source event.
> 
> For example, imagine you've added this file hash indicator to Timeline:
> 
> `threat.indicator.file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`
> 
> A new Timeline opens with an automatically populated KQL query. The query contains the indicator field value pair (mentioned previously) and the mapped source event field value pair, which is:
> 
> `file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.
> 
> Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field and value pairs.

Reviewer 3: The "pairs" here get called both "field and value pairs" and "field value pairs," but I think "[field-value pairs](https://en.wikipedia.org/wiki/Name%E2%80%93value_pair)" is more standard. 
```suggestion
When you add an indicator to Timeline, a new Timeline opens with a pre-populated KQL query. The query contains the indicator field-value pair that you selected plus the field-value pair of the mapped source event.

For example, imagine you've added this file hash indicator to Timeline:

`threat.indicator.file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`

A new Timeline opens with an automatically populated KQL query. The query contains the indicator field-value pair (mentioned previously) and the mapped source event field-value pair, which is:

`file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.

Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field-value pairs.
```
